src/Compilers/CSharp/Test/Semantic/Semantics/SwitchTests.cs

```diff
@@ -2949,6 +2949,482 @@ public static void Main()
                 Diagnostic(ErrorCode.WRN_UnreachableCode, "break").WithLocation(14, 17));
         }
 
+        [Fact, WorkItem(47878, "https://github.com/dotnet/roslyn/issues/47878")]
+        public void Bug47878()
+        {
+            var text = @"
+using System;
+public class C
+{
+    public static void Main()
+    {
+        int x1 = (1, 2, 3, 4, 5, 6, 7, 8, 9, 10) switch
+        {
+            (1, 2, 3, 4, 5, 6, 7, 8, 9, 10) => 1,
```

Comment:
Can we add a negative case here, basically a literal that won't match. Just for completeness.